extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1138,16 +1138,26 @@ export class Bind {
 
       updates.forEach(update => {
         const {boundProperty, newValue} = update;
+        const {property} = boundProperty;
         const mutation = this.applyBinding_(boundProperty, element, newValue);
+
         if (mutation) {
           mutations[mutation.name] = mutation.value;
-          const {property} = boundProperty;
           if (property == 'width') {
             width = isFiniteNumber(newValue) ? Number(newValue) : width;
           } else if (property == 'height') {
             height = isFiniteNumber(newValue) ? Number(newValue) : height;
           }
         }
+
+        if (isPropertyAFormValue(element.tagName, property)) {
+          const dispatchAt =
+            element.tagName === 'OPTION' ? element.closest('SELECT') : element;
```

Comment:
Out of curiosity, why is this necessary? Also use `closestAncestorElementBySelector()` helper in dom.js in case `Element.closest` is not available.

Reply:
Thanks! The native `InputEvent` is also dispatched at the parent `<select>` when the selected `<option>` changes. I think it makes sense, since it is the value of the `<select>` element that is changed.

I just added a comment explaining this!